Feature/documentation

wvpelt opened the pull request.

Commit:
Added example for DRYResourceReader

DRYResourceReader:
- Fixed typo in DRYPlistResourceReader
- Added extra documentation to README.md file
- Added example for DRYResourceReader

Commit:
Updated README for configuration utilities

Commit:
Updated documentation

Commit:
Added extra documentation for foundation utilities

mikeseghers commented:
Great work up till now, have you gone through all classes and categories yet? Have you looked into test coverage?

Commit:
Documentation, example app, new DRY utilities

- More documentation is added
- Example app added which includes examples of most the utilities
- Added UIColor+DRYUtil
- Added UIImage+ColorPixel

Commit:
project file update

Commit:
Added extra test for UIColor+DRYUtil

Commit:
Updated readme and added example for UIColor

Commit:
Minor readme updates

Commit:
Fixed missing imports

Commit:
Fixed imports for UIImage+ColorPixel

Commit:
Added import

Commit:
added another missing import

Commit:
Fixed travis link

Commit:
added reference to example usages

Commit:
Added documentation for security and CoreData utilities

Commit:
Fixed typo's in readme

Commit:
fixed typo

Commit:
Extended documentation on resource readers, changed some phrasing in example screens, and added json resource reader, together with a dictionary resource reader.

Commit:
Doc and cleanup

Commit:
Removed some warnings, improved naming of init methods with fallback resource reader, changed read to showcase fallback
